apps/yara/CMakeLists.txt

```diff
@@ -14,22 +14,14 @@ message (STATUS "Configuring apps/yara")
 # Dependencies
 # ----------------------------------------------------------------------------
 
-# Disable older MSVC.
-if (MSVC AND MSVC_VERSION LESS 1800)
-    message (STATUS "  Visual Studio 2012 or newer required to build yara.")
-    return ()
-endif ()
-
-# Disable parallelism in MSVC as it supports only OpenMP 2.0.
-if (NOT MSVC)
```

Comment:
Excellent! Then you have so much free time to change it during the trip back :trollface: 

Okay seriously, I don't like this solution either, so you can disable MSVC for now. But my questions are:
* Where and how do you do ```find_package``` now? Are OpenMP flags added to all targets?
* Why don't you make Windows/ICC builds a priority? Windows/MSVC builds suck because of OpenMP 2.0...